admissions/admissions/models.py

```diff
@@ -30,8 +30,10 @@ def is_member_of_webkom(self):
         """
         Return whether the user is a member of the webkom-group or not
         """
-        webkom = Group.objects.get(name=constants.WEBKOM_GROUPNAME)
-        return Membership.objects.filter(user=self, group=webkom).exists()
+        webkom = Group.objects.filter(name=constants.WEBKOM_GROUPNAME)
+        if not webkom.exists():
+            return False
+        return Membership.objects.filter(user=self, group=webkom.first()).exists()
```

Comment:
Not sure I'm following this

Reply:
Ah, the reason I changed this part, is that when the tests are running - there is no group named `webkom` unless it is created in the tests. And by running `Group.objects.get(...)` django will throw notfound errors if it is not found, but with filter I can handle it more gracefully (though I suppose a try catch may make it clearer what is going on)

And I wanted to change this rather than creating a webkom-group for every test-case because it feels neater, and I see no reason to require webkom as a group to run the project in my search for improved genericness.

Reply:
Updated to a try-catch with a clarifying comment now